storage-proofs/src/vde.rs

```diff
@@ -91,24 +91,25 @@ where
 pub fn decode_domain_block<H>(
     sloth_iter: usize,
     replica_id: &H::Domain,
-    data: &[H::Domain],
+    byte_data: &[u8],
     node: usize,
+    node_data: <H as Hasher>::Domain,
     parents: &[usize],
 ) -> Result<H::Domain>
 where
     H: Hasher,
 {
-    let key = create_domain_key::<H>(replica_id, node, parents, data)?;
+    let key = create_domain_key::<H>(replica_id, node, parents, &byte_data)?;
```

Comment:
this change makes `create_domain_key` the same as `create_key` I believe, so you can just use that

Reply:
Ok, I'll take a look, I wasn't sure why there were two functions now.

Reply:
Yes, you're right, I removed `create_domain_key` which is not used anymore.